packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -235,4 +238,5 @@ Contains breaking changes
 [code_magic_url]: https://codemagic.io/start/
 [example_project_readme_doc_url]: https://github.com/Prime-Holding/rx_bloc/blob/master/packages/rx_bloc_cli/example/README.md
 [golden_tests_doc_url]: https://github.com/Prime-Holding/rx_bloc/blob/master/packages/rx_bloc_cli/example/docs/golden_tests.md
-[mfa_doc_url]:https://github.com/Prime-Holding/rx_bloc/blob/master/packages/rx_bloc_cli/example/docs/mfa.md
\ No newline at end of file
+[mfa_doc_url]:https://github.com/Prime-Holding/rx_bloc/blob/master/packages/rx_bloc_cli/example/docs/mfa.md
+[patrol_integration_test_url]:https://github.com/Prime-Holding/rx_bloc/blob/master/packages/rx_bloc_cli/mason_templates/bricks/patrol_integration_tests/__brick__/docs/README.md
```

Comment:
Please use the actual file name  `__brick__/docs/patrol_integration_test.md` instead of `__brick__/docs/README.md`

Reply:
fixed